Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/SettingsTab/SettingsView.xaml.cs

```diff
@@ -1,12 +1,54 @@
-﻿using System.Windows.Controls;
+﻿using System.Windows;
+using System.Windows.Controls;
 
 namespace Smart_Pacifier___Tool.Tabs.SettingsTab
 {
     public partial class SettingsView : UserControl
     {
+        private bool isUserMode = true;  // Default to User Mode
```

Comment:
This will only save the state when on the page, once the view is changed, the bool is not saved.

Reply:
@VincentStriegel Please Check now :)